### DISCUSSION TITLE
Cross platform support

### DISCUSSION THREAD

**Discussion by lucsoft** [FluxpointDev/MultiRPC#17] [Old]

**Comment by xXBuilderBXx** [comment]
This is planned and one of the devs that helps me Azy is working hard to add support, thanks for using the app and i will keep this open for now.

**Comment by xXBuilderBXx** [comment]
Hi thanks for using MultiRPC we now have support for it using Discord games, this will allow us to add multiplatform support and better updating.
You can request a gift code from me Builderb#0001 or join the support server https://discord.gg/TjF6QDC

**Comment by Azyyyyyy** [comment]
Progress on a cross-platform version will be in the [rewrite](https://github.com/FluxpointDev/MultiRPC/tree/rewrite) branch, when it is stable enough to be used we'll tag you in this issue 🐱‍💻

**Comment by xXBuilderBXx** [comment]
@lucsoft a beta rewrite version is now available for linux you can try it out with build release 7.

**Comment by Azyyyyyy** [comment]
Going to lock as this is now the case and in a stable(ish) state. Any bugs you encounter with the rewrite should be made into a new issue 👍

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
